eng/versioning/version_client.txt

```diff
@@ -246,7 +246,7 @@ com.azure.resourcemanager:azure-resourcemanager-compute;2.26.0;2.27.0
 com.azure.resourcemanager:azure-resourcemanager-containerinstance;2.26.0;2.27.0
 com.azure.resourcemanager:azure-resourcemanager-containerregistry;2.26.0;2.27.0
 com.azure.resourcemanager:azure-resourcemanager-containerservice;2.26.0;2.27.0
-com.azure.resourcemanager:azure-resourcemanager-cosmos;2.26.0;2.27.0
+com.azure.resourcemanager:azure-resourcemanager-cosmos;2.26.0;2.28.0-beta.1
```

Comment:
@XiaofeiCao appears correct as 2.28.0-beta.1, but seems a bit strange as it is 1.26 vs 1.28
Could you confirm?

Reply:
Yeah, 2.27.0;2.28.0-beta.1 is more appropriate. Other versions are fine to me.

Yet previous version is used for detecting breaking changes with current version, and for preview, we don't mind breaking changes as long as they don't break our code(compilation error). So I think it should be fine?

Reply:
OK. Then no need to change here.